packages/form-validation/src/validation.js

```diff
@@ -252,7 +252,20 @@ export default class AuroFormValidation {
        * The validityState definitions are located at https://developer.mozilla.org/en-US/docs/Web/API/ValidityState.
        */
 
-      let hasValue = elem.value && elem.value.length > 0;
+      let hasValue = false;
+
+      // Check string for having a value
+      if (typeof elem.value === "string") {
+        hasValue = elem.value && elem.value.length > 0;
```

Comment:
```suggestion
        hasValue = !!elem.value;
```
nit: can be simplified like this.

Reply:
I think keeping this more explicit to it's purpose (checking that the length is longer than a minimum threshold) is good here.

If ESLint would let me, I'd prefer to have it written as:

```javascript
hasValue = elem.value?.length > 0;
```

ESlint also won't let us use the double-not operator anyways, which is sad, because I'm a big fan.